### PR TITLE
Show the tooltip only on the disabled button

### PR DESCRIPTION
**Pull request by JuliaEvseeva**
This PR shows the tooltip only on the disabled button and hides it on the enabled button.

### CODE REVIEW

**Comment by JuliaEvseeva** [comment]
@andreySadomovskiy, PTAL.

File: js/pricing.js [hunk 1]
```diff
@@ -85,16 +89,19 @@ $(
          * Changes element's disabled/enabled state.
          *
          * @param {jQuery} element the element to change the state for
+         * @param {jQuery} elementHolder the element holder to apply or hide a tooltip
          * @param {boolean} enable denotes whether the element should be enabled.
          * If `true` — enables the element, otherwise — disables
          */
-        function changeElementState(element, enable) {
+        function changeElementState(element, elementHolder, enable) {
```
Comment (andreySadomovskiy) [inline js/pricing.js:96]: I'd rather use only `element` parameter, but pass to the function the `orderButtonHolder `. And then find the button inside.

**Review by andreySadomovskiy — COMMENTED** [review]
@JuliaEvseeva, please  PTAL at the comment below.

File: js/pricing.js [hunk 2]
```diff
@@ -90,11 +94,13 @@ $(
          */
         function changeElementState(element, enable) {
             if (enable) {
-                element.removeAttr('disabled');
-                element.removeClass('disabled');
+                element.find($orderButton).removeAttr('disabled');
```
Comment (andreySadomovskiy) [inline js/pricing.js:97]: Use `const` for the `element.find($orderButton)`, please.

**Review by andreySadomovskiy — APPROVED** [review]
@JuliaEvseeva , PTAL at the comment bellow and LGTM.

**Pull request merged**